lib/tasks/update_spreadsheets.rake

```diff
@@ -2,5 +2,6 @@ namespace :spreadsheets do
   desc 'Updates Google spreadsheets with the latest audit data'
   task update: :environment do
     AddAuditDataToSpreadsheetJob.perform_later('vacancies')
+    AddFeedbackToSpreadsheetJob.perform_later
```

Comment:
I wonder why we have a new job here as the existing job above looks to be configured to be reusable by the acceptance of the parameter. Could we save on an extra class here?

```
AddAuditDataToSpreadsheetJob.perform_later('feedback')
```

Reply:
I've given the commits here a read through and now understand that we've refactored this to allow us with `AuditData` and other sources, like `Feedback`. 

Should we remove this line https://github.com/dxw/teacher-vacancy-service/blob/develop/app/models/audit_data.rb#L9 from this commit https://github.com/dxw/teacher-vacancy-service/pull/766/commits/09572933748c04e1e1baaa720128117b61e830aa which looks to have set an array of categories a bit prematurely.

Reply:
I've removed the feedback category from that commit 👍